src/main/io/msp.c

```diff
@@ -942,12 +942,8 @@ static int processOutCommand(mspPacket_t *cmd, mspPacket_t *reply)
 
         case MSP_LED_STRIP_CONFIG:
             for (int i = 0; i < LED_MAX_STRIP_LENGTH; i++) {
-                ledConfig_t *ledConfig = ledConfigs(i);
```

Comment:
Original idea was to separate internal (optimized) representation and (quite generic) wire format